src/background/tabReloader.js

```diff
@@ -36,7 +36,11 @@ export class TabReloader extends Component {
   }
 
   async init() {
-    this.proxyHandler.lastChangedOrigin.subscribe(TabReloader.onOriginChanged);
+    this.proxyHandler.lastChangedOrigin.subscribe((origin) => {
+      if (origin) {
```

Comment:
Subscriptions always trigger with the default value with a microtask, if we don't filter out empty origins we will always reload all tabs